test/development/app-dir/dynamic-io-dev-errors/dynamic-io-dev-errors.test.ts

```diff
@@ -68,7 +68,7 @@ describe('Dynamic IO Dev Errors', () => {
         `See more info: https://nextjs.org/docs/messages/next-prerender-missing-suspense` +
         '\n    at Page [Server] (<anonymous>)' +
         // TODO(veil): Should be ignore-listed. Feel free to adjust the component name since it's Next.js internals.
-        '\n    at InnerLayoutRouter (' +
+        '\n    at parallelRouterKey (' +
```

Comment:
sourcemap bug. will file this separately against SWC.

Reply:
Current thinking is that it's not a SWC bug since it doesn't map to the destructured param name in a minimal example. Likely off-by-one error.